bagisto/bagisto
# Issue Fixed #7073

## Issue Reference

#7073 
## Description

Updated Style of page Header
## How To Test This?

Go to the Admin Panel
Try to create a new Product, category, attribute etc.
Check the same concern for all other sections in the Admin Panel.
## Documentation
